test/unit/lib/redmine/menu_manager/menu_helper_test.rb

```diff
@@ -100,14 +100,14 @@ def test_render_menu_node_with_nested_items
 
     html_node = HTML::Document.new(@response.body)
     assert_select(html_node.root, "li") do
-      assert_select("a.parent-node", "Parent node")
+      assert_select("a.parent-node-menu-item", "Parent node")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.